config-templates/ibm/ibm-csi-block/1.4.0/storage-class-template.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "{{ name }}"
```

Comment:
Can you add annotation for is-default-class in storageclass-template ?
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: "{{ name }}"
  annotations:
  storageclass.kubernetes.io/is-default-class: "{{ is-default-class }}"

Reply:
done